specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2024-01-01-preview/Recommendations.json

```diff
@@ -208,79 +235,52 @@
           "description": "Description of the recommendation.",
           "type": "string"
         },
-        "recommendationTypeTitle": {
-          "description": "Title of the recommendation type.",
-          "type": "string"
-        },
-        "recommendationTypeId": {
-          "description": "Id of the recommendation type.",
-          "type": "string"
-        },
-        "category": {
-          "description": "Category of the recommendation.",
-          "$ref": "#/definitions/RecommendationCategory"
+        "creationTimeUtc": {
+          "description": "The time stamp (UTC) when the recommendation was created.",
+          "type": "string",
+          "format": "date-time"
         },
-        "context": {
-          "description": "Context of the recommendation.",
-          "$ref": "#/definitions/RecommendationContext"
+        "lastEvaluatedTimeUtc": {
+          "description": "The time stamp (UTC) when the recommendation was last evaluated.",
+          "type": "string",
+          "format": "date-time"
         },
-        "workspaceId": {
-          "description": "Id of the workspace this recommendation refers to.",
+        "lastModifiedTimeUtc": {
+          "description": "The time stamp (UTC) when the recommendation was last modified.",
           "type": "string",
-          "format": "uuid"
+          "format": "date-time"
         },
-        "actions": {
-          "description": "List of actions to take for this recommendation.",
+        "suggestions": {
+          "description": "List of suggestions to take for this recommendation.",
           "type": "array",
           "items": {
-            "$ref": "#/definitions/RecommendedAction"
+            "$ref": "#/definitions/RecommendedSuggestion"
           },
           "x-ms-identifiers": []
         },
-        "state": {
-          "description": "State of the recommendation.",
-          "$ref": "#/definitions/RecommendationState"
-        },
-        "priority": {
-          "description": "Priority of the recommendation.",
-          "$ref": "#/definitions/RecommendationPriority"
-        },
-        "lastEvaluatedTimeUtc": {
-          "description": "The time stamp (UTC) when the recommendation was last evaluated.",
-          "type": "string",
-          "format": "date-time"
-        },
-        "hideUntilTimeUtc": {
-          "description": "The time stamp (UTC) when the recommendation should be displayed again.",
+        "resourceId": {
+          "description": "Id of the resource this recommendation refers to.",
           "type": "string",
-          "format": "date-time",
           "x-nullable": true
         },
-        "displayUntilTimeUtc": {
-          "description": "The timestamp (UTC) after which the recommendation should not be displayed anymore.",
-          "type": "string",
-          "format": "date-time",
+        "additionalProperties": {
```

Comment:
@xuhumsft - Following up on the recent issue we had in Analytics with 'additionalProperties'
Is it fine to have a property named additionalProperties? or should we change it?
despite that it was here already.

@itielMaimon - FYI - in case we need to ask for exemption here or change it, see this: https://github.com/Azure/azure-openapi-validator/blob/main/docs/avoid-additional-properties.md

Reply:
Fixed the Swagger ModelValidation, also, I don't see we have a Lint Diff Error.